PRD-SIMPLE.md

```
@@ -0,0 +1,39 @@
+# Erin's Escapades - Simple PRD
+
+## One Sentence
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)